.azure/pipelines/helix-matrix.yml

```diff
@@ -1,5 +1,9 @@
 # We only want to run full helix matrix on main
-pr: none
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - '*'
```

Comment:
I remember asking about a PR that switched to `pr: none` somewhere but we shouldn't be running this pipeline for all PRs